### ISSUE TITLE
Information about new releases from the marked artists 

### ISSUE THREAD

**Issue by siggi1984** [fast4x/RiMusic#2288]
### Description

I don't know if this is possible, but is there a way to be notified in some way about new releases from the artists you have tagged? Be it a notification or a hint through a dot or something similar.

In any case, this developer here has achieved this.

https://github.com/maxrave-dev/SimpMusic

I think such a function would be a great addition to this APP and you would be up to date with new releases from your favorite artists.

Thank you for the great app and your hard work.

### Suggested Solution

X

**Comment by fast4x** [comment]
It's just implemented, look at quick pics and enable from relative settings

**Comment by siggi1984** [comment]
> It's just implemented, look at quick pics and enable from relative settings ![Screenshot_2024-06-14-01-18-52-821_it.fast4x.rimusic-edit.jpg](https://private-user-images.githubusercontent.com/1484476/339578241-f3dfacdd-135e-41c9-b4a2-221b9422fd03.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgzNTM4OTEsIm5iZiI6MTcxODM1MzU5MSwicGF0aCI6Ii8xNDg0NDc2LzMzOTU3ODI0MS1mM2RmYWNkZC0xMzVlLTQxYzktYjRhMi0yMjFiOTQyMmZkMDMuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYxNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MTRUMDgyNjMxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzE2MzBjYmM1ZWQ2Y2Q0NzMxY2M5Y2YxZTdjZjcxZDJhZjYwY2ZlMTkyOTU4YjY1NjIzYzc2ZTQyNjZhNGQ3YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.ztVRnYTUil_SHYo3YdTLMGhPVbUIXddEzdm0hVRFi2I)

Yes, I know this function, but new releases are generally shown here and if you have tagged one of these artists, their release is highlighted, but that's not what I mean. I would like to be notified when one of my tagged artists releases a new single or album. Just like with SimpMusic. Maybe you can take a look at it.

**Comment by fast4x** [comment]
Push notification isn't possible

**Comment by siggi1984** [comment]
> Push notification isn't possible

Okay, it's not about the push notification but about the info that one of my favorite artists has a new single or album.

**Comment by fast4x** [comment]
How would you like to receive the notification?

**Comment by siggi1984** [comment]
> How would you like to receive the notification

could possibly be displayed on the homepage.

**Comment by fast4x** [comment]
Inside app?

**Comment by siggi1984** [comment]
> Inside app?

yes I mean in the app the standard page

**Comment by siggi1984** [comment]
or a kind of history of publications, similar to the listening history

**Comment by ikanakova** [comment]
Please make it optional, because my favorite artists don't release anything anymore because they're mostly dead...

**Comment by siggi1984** [comment]
Wouldn't it be possible to adopt the implementation from SimpMusic?

**Comment by fast4x** [comment]
I'll look in

**Comment by siggi1984** [comment]
> I'll look in

With this function we would be independent of Spotify and YTM.

**Comment by siggi1984** [comment]
bro, what do you think, would this feature request be possible in the near future, but I don't want to put any pressure on you now.
